pages/garden/index.js

```diff
@@ -527,10 +507,19 @@ export default function Garden({
             $alive={activePet.alive}
             $animationtype={animationState}
           >
-            {activePet.alive ? activePet.picture : "☠"}
+            {activePet.alive ? (
+              <AnimatedPet
+                pet={activePet.animations}
+                dying={activePet.dying}
+                movingSpeedFactor={characteristicEffects.speedFactor}
+                setPetCollection={setPetCollection}
```

Comment:
I did this for the handling of dead pets but I suppose it would also make sense to do this for the big indicator update function in the garden.js. I will update that function in the coming refactoring story as it's not directly connected to the current animation story.